### PR TITLE
Forms active

### PR DESCRIPTION
**Pull request by teddy4445**
change the design and logic according to the instructions from the design team for only one page.

Need help with the "X" and "V" on the right of the input

### CODE REVIEW

File: src/client/public/main.css
Comment (Hadari23) [inline src/client/public/main.css:798]: We repeat too much of the code. This is also a problem that I have to consider. I think that this is not necessary and we need to create basic (generic) submit button style and on specific styles we should change only the properties that really changes (for example: border radius ahouldn't change among different styles of the button, hence, we shouldn't set this property more than once). As well as other properties.

File: src/client/public/sign-for-beta.html
Comment (Hadari23) [inline src/client/public/sign-for-beta.html:100]: Don't use "style" in HTML tags!!!
Preferably, add classes and edit style in main.css

File: src/client/public/sign-for-beta.html
Comment (Hadari23) [inline src/client/public/sign-for-beta.html:98]: I have another version of this form in my branch. We should decide which one to choose...

File: src/client/public/sign-for-beta.html
Comment (Hadari23) [inline src/client/public/sign-for-beta.html:108]: We better use indicative ids reather than "input1","input2", etc..
for example: id="institution"

File: src/client/public/sign-for-beta.js
Comment (Hadari23) [inline src/client/public/sign-for-beta.js:2]: I think it's OK but you should consider the changes in the HTML as I wrote above.

**Review by Hadari23 — CHANGES_REQUESTED** [review]
אחלה, יש כמה דברים שצריך להתחשב בהם

File: src/client/public/main.css
Comment (teddy4445) [reply src/client/public/main.css:798]: I agree. 
From project perspctive, the additinal value of a better CSS code is really small while we do not have all pages ready... 
I think it will be fair to open an issue regarding this but acpet the situation as is right now

Comment (teddy4445) [reply src/client/public/main.css:798]: https://github.com/teddy4445/lecture_website_app_generator/issues/28

File: src/client/public/sign-for-beta.html
Comment (teddy4445) [reply src/client/public/sign-for-beta.html:98]: I will replace to your version after merge

File: src/client/public/sign-for-beta.html
Comment (teddy4445) [reply src/client/public/sign-for-beta.html:108]: I thought about it, this is done on the "name" attr but here I used this for the for loop in the JS

File: src/client/public/sign-for-beta.js
Comment (teddy4445) [reply src/client/public/sign-for-beta.js:2]: I will, this file is per page...

**Pull request merged**
